packages/boot/tools/ibc/mocks.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * mock bridgeInbound events for ICA (ICS-27) flow
+ * see [./ics27-1.md](./ics27-1.md) for more details
+ */
+export const icaMocks = {
+  startChannelOpenInit: {
+    // ICA Channel Creation
+    channelOpenAck: obj => ({
+      /** sourced from e2e testing with sim chains */
+      type: 'IBC_EVENT',
+      blockHeight: 99,
+      blockTime: 1711571357,
+      event: 'channelOpenAck',
+      portID: obj.packet.source_port,
+      channelID: 'channel-0',
+      counterparty: {
+        port_id: obj.packet.destination_port,
+        channel_id: 'channel-1',
+      },
+      counterpartyVersion:
+        '{"version":"ics27-1","controllerConnectionId":"connection-0","hostConnectionId":"connection-0","address":"osmo1234","encoding":"proto3","txType":"sdk_multi_msg"}',
+      connectionHops: obj.hops,
+    }),
+    // XXX channelOpenAckFailure
+  },
+  sendPacket: {
+    // ICA Send Packet (Transaction)
+    acknowledgementPacket: _obj => ({
```

Comment:
This step is not reached in tests yet, only `channelOpenAck. channelOpenAck`

Reply:
thanks for the note. when do you plan to cover the case? that would be good to document in the code.